wenab-app/tsconfig.json

```diff
@@ -13,14 +13,23 @@
     "strict": true,
     "forceConsistentCasingInFileNames": true,
     "noFallthroughCasesInSwitch": true,
-    "module": "esnext",
+    "module": "es2022",
```

Comment:
why `es2021` in some places and `es2022` here?

Reply:
Copy pasta from Froggie 🙃

Reply:
- [ ] consolidate `es2021` and `es2022` refs